db/testing.py

```diff
@@ -28,36 +28,38 @@ def reset_db(self):
-        run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_primary_keys.sql'))
-        run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_foreign_keys.sql'))
-        run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_indexes.sql'))
+        db.run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_types.sql'))
+        db.run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_tables.sql'))
+        db.run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_primary_keys.sql'))
+        db.run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_foreign_keys.sql'))
+        db.run_sql_script(os.path.join(ADMIN_SQL_DIR, 'create_indexes.sql'))
 
     def drop_tables(self):
-        with create_cursor() as cursor:
+        with db.engine.connect() as connection:
             # TODO(roman): See if there's a better way to drop all tables.
-            cursor.execute('DROP TABLE IF EXISTS highlevel_json       CASCADE;')
-            cursor.execute('DROP TABLE IF EXISTS highlevel            CASCADE;')
-            cursor.execute('DROP TABLE IF EXISTS lowlevel             CASCADE;')
-            cursor.execute('DROP TABLE IF EXISTS statistics           CASCADE;')
-            cursor.execute('DROP TABLE IF EXISTS incremental_dumps    CASCADE;')
-            cursor.execute('DROP TABLE IF EXISTS dataset_eval_jobs    CASCADE;')
-            cursor.execute('DROP TABLE IF EXISTS dataset_class_member CASCADE;')
-            cursor.execute('DROP TABLE IF EXISTS dataset_class        CASCADE;')
-            cursor.execute('DROP TABLE IF EXISTS dataset              CASCADE;')
-            cursor.execute('DROP TABLE IF EXISTS "user"               CASCADE;')
-        commit()
+            connection.execute('DROP TABLE IF EXISTS highlevel_json       CASCADE;')
+            connection.execute('DROP TABLE IF EXISTS highlevel            CASCADE;')
+            connection.execute('DROP TABLE IF EXISTS lowlevel             CASCADE;')
+            connection.execute('DROP TABLE IF EXISTS statistics           CASCADE;')
+            connection.execute('DROP TABLE IF EXISTS incremental_dumps    CASCADE;')
+            connection.execute('DROP TABLE IF EXISTS dataset_eval_jobs    CASCADE;')
+            connection.execute('DROP TABLE IF EXISTS dataset_class_member CASCADE;')
+            connection.execute('DROP TABLE IF EXISTS dataset_class        CASCADE;')
+            connection.execute('DROP TABLE IF EXISTS dataset              CASCADE;')
+            connection.execute('DROP TABLE IF EXISTS "user"               CASCADE;')
 
     def drop_types(self):
-        with create_cursor() as cursor:
-            cursor.execute('DROP TYPE IF EXISTS eval_job_status CASCADE;')
-        commit()
+        with db.engine.connect() as connection:
+            connection.execute('DROP TYPE IF EXISTS eval_job_status CASCADE;')
+
+    def data_filename(self, mbid):
+        """ Get the expected filename of a test datafile given its mbid """
```

Comment:
Do you want me to complain about docstring formatting for the sake of consistency? I can ignore that if you don't want to worry about it. I mostly use https://google-styleguide.googlecode.com/svn/trunk/pyguide.html?showone=Comments#Comments as a reference. IDE/pylint reminds me about this stuff most of the time, so I kind of got used to the style.